.github/workflows/wildfly-it-tests-bootable.yml

```
@@ -0,0 +1,40 @@
+name: WildFly Bootable JAR Integration Tests
+
+on:
+  push:
+    branches: [ 'master' ]
+  pull_request:
+    branches: [ 'master' ]
+
+jobs:
+  wildfly-it-tests:
+    runs-on: ${{ matrix.os }}
```

to keep feature parity between both CI runs?
```suggestion
    if: '! github.event.pull_request.draft'
    runs-on: ${{ matrix.os }}
```

---

Sure, that was for debugging reasons while keeping the draft.